readme.md

```diff
@@ -238,6 +238,14 @@ Default: `false`
 
 Disable CSS [animations](https://developer.mozilla.org/en-US/docs/Web/CSS/animation) and [transitions](https://developer.mozilla.org/en-US/docs/Web/CSS/transition).
 
+##### javascriptEnabled
+
+Type: `boolean`<br>
+Default: `true`
+
+Options to enable javascript execution on the page.
+> nb: If disabled, it also disable `modules` and `scripts`.
```

Comment:
Are you sure these will not work? The Puppeteer docs are not clear on this point. You'll need to do some testing. Would be nice if they would work. The point is to disable the websites JS, not the user's injected JS. We might have to do some code changes to make it work.

Reply:
The documentation is only said about stopping the javascript execution after the options is set.

I've tried on unit test for this one, and found that `script` options still executed, while using `module` isn't.

Is it good enough? or should it re enabled after page is loaded, right before inject script / module?

Reply:
> I've tried on unit test for this one, and found that script options still executed, while using module isn't.

Seems weird that `scripts` works, but not `modules`. Are you sure you tested it correctly? I think you need to look more into this.

Reply:
Oops, I think I messed up. Because of the typo, the result is `scripts` and `modules` not works when this options set to `false`.

As `modules` and `scripts` could be work now by resetting the options after page is loaded. What do you think?